README.md

```diff
@@ -98,16 +98,16 @@ Supported extensions: .appx, appxupload or .zip
 
 ### UWP Bundles
 ```
-Supported extensions: .appxbundle, appxupload or .zip
+Supported extensions: .appxbundle, .appxupload, .msixbundle, .msixupload or .zip
 ```
 
-> For .appxbundle app packages, the manifest is checked for the correct name of the appx subpackage. If it is found, the metadata is parsed directly from there, otherwise we scavenge for as much as we can get from the appxbundle manifest and metadata
+> For .appxbundle or .msixbundle app packages, the manifest is checked for the correct name of the appx subpackage. If it is found, the metadata is parsed directly from there, otherwise we scavenge for as much as we can get from the appxbundle/msixbundle manifest and metadata
```

Comment:
According to this page: 
https://mspoweruser.com/microsoft-announces-new-msix-app-packaging-format/
```
This new MSIX installer will support all type of Windows applications including Win32, WPF, WinForm and UWP.
```

Will this change support UWP apps only? If so, can you please document it?

Reply:
I'm not sure to be honest.  I'll try to reach out and document this if necessary